File: VDS_Frontend/src/app/consulta/Models/consulta.dto.ts
export interface Consulta {
  id?: number;
  fechaConsulta: string;
  motivo: string;
  notas?: string;
  medicacion?: string;
  mascotaId: number;
  veterinarioId: number;
  clinicaId?: number;
  pruebaIds: number[];
  vacunaIds: number[];
}
  

File: VDS_Backend/src/main/java/com/uoc/tfm/vds_backend/consulta/dto/ConsultaDTO.java
package com.uoc.tfm.vds_backend.consulta.dto;

import java.time.LocalDateTime;
import java.util.List;

import lombok.Data;

@Data
public class ConsultaDTO {
    private Long id;
    private LocalDateTime fechaConsulta;
    private String motivo;
    private String notas;
    private String medicacion;
    private Long mascotaId;
    private Long veterinarioId;
    private List<Long> pruebaIds;
    private List<Long> vacunaIds;
}
